src/main/java/us/abstracta/wiresham/Flow.java

```diff
@@ -88,7 +88,8 @@ public static Flow fromWiresharkJsonDump(File file, String serverAddress)
               Long.parseLong(layers.at(WIRESHARK_TIME_DELTA_PATH).asText().replace(".", ""))
                   / 1000000;
           return isServerAddress(sourceIp, sourcePort, serverAddress) ? new SendPacketStep(
-              hexDump, timeDeltaMillis) : new ReceivePacketStep(hexDump);
+              hexDump, timeDeltaMillis, Integer.parseInt(sourcePort))
+              : new ReceivePacketStep(hexDump);
```

Comment:
y el puerto del receive?

Reply:
Do we want to determine the client port? Keep in mind that if we do, when the user runs the flow it will wait for a connection in client port specified in yaml which won't be the same as the server port.
The `sourcePort` defined over there is the client port that belongs to the connection stablished. Therefore, is assigned dynamically. Do we want to set it in stone?